templates/account/account.py

```diff
@@ -92,60 +105,92 @@ def _authorize_users(self, account):
             account.unauthorize_user(username=user)
 
     def uninstall(self):
+        self.state.check('actions', 'install', 'ok')
         cl = self.ovc
         acc = cl.account_get(self.name)
         acc.delete()
```

Comment:
Need to remove the install state here.

Reply:
why?

Reply:
cause if you don't do it, you can never execute the install action anymore for example.
And even from a concept point of view, you can't have a `state install OK` when this is actually not the case in reality